cloudwatch_metrics_collector/launch/cloudwatch_metrics_collector.launch.py

```diff
@@ -45,16 +51,12 @@ def generate_launch_description():
     # ID as a default dimension
     if os.environ[RoboMakerEnvVariables.CAPABILITY] == "AWSRoboMakerSimulation" and RoboMakerEnvVariables.SIMULATION_JOB_ID in os.environ:
        aws_default_metric_dimensions = "SimulationJobId:" + os.environ.get(RoboMakerEnvVariables.SIMULATION_JOB_ID)
-
-  
       
   parameters = [launch.substitutions.LaunchConfiguration(CONFIG), ]
-  if aws_robomaker_metric_namespace:
-    parameters.append({"aws_metrics_namespace": aws_robomaker_metric_namespace})
   if aws_default_metric_dimensions:
     parameters.append({"aws_default_metric_dimensions": aws_default_metric_dimensions})
-      
-
+  parameters.append({"aws_metrics_namespace": launch.substitutions.LaunchConfiguration("aws_metrics_namespace")})
```

Comment:
There are 4 sources for aws_metrics_namespace: default_config, user provided config, launch argument, env variable. 
I think the order we want to look at these is:
1) launch argument
2) environment variable (not sure where this one fits in)
2) user provided config
3) default config

I think right now this is going to get
1) Env variable
1) launch argument
2) default config
3) user provided config

and since the default config always exists it will never get the user provided config